frontend/jupyter/src/app/pages/notebook-page/overview/overview.component.ts

```diff
@@ -151,6 +151,17 @@ export class OverviewComponent implements OnDestroy {
     }
   }
 
+  get notebookCreator(): string {
+    return this.getNotebookCreator(this.notebook);
+  }
+
+  getNotebookCreator(notebook: NotebookRawObject): string {
+    if (!notebook?.metadata?.annotations) {
+      return null;
+    }
+    return notebook.metadata.annotations['notebooks.kubeflow.org/creator'];
```

Comment:
What is that?...

Reply:
seems like a new annotation. Already had a comment in the ticket about it. https://jirab.statcan.ca/browse/BTIS-668?focusedId=3344374&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3344374

Way i see it, no harm now. Will see with the notebook controller updated if it adds that label. Or if the new backend adds it on notebook creation.